src/api/router.go

```diff
@@ -25,6 +25,7 @@ func NewRouter(c GameController) *mux.Router {
 	api.HandleFunc("/game/{gameID}", c.UpdateGame).Methods("PATCH")
 	api.HandleFunc("/game/{gameID}", c.ResetGame).Methods("DELETE")
 	api.HandleFunc("/game/{gameID}/start", c.StartRound).Methods("PUT") // initiates a new round with default values
+	api.HandleFunc("/game/{gameID}/randomGame", c.StartRandomGame).Methods("GET")
```

Comment:
I was re-reading some of the guidelines for building a RESTful API recently (https://restfulapi.net/http-methods/ is a good resource!) and wonder if we should make this a `PUT` method instead of a `GET`? It looks like `GET` is meant to be idempotent and only retrieve data (not add/alter anything in the db)

Reply:
Thanks... I'm not very familiar with RESTful APIs yet, so I wasn't really sure which request to use.